### PR TITLE
chore: Add a simple lambda that calls get for performance testing

### PR DESCRIPTION
**Pull request by nand4011**
Add an example lambda, simple-get, that calls get on a momento cache repeatedly with a small delay in between. It is used for latency testing in lambda.

Correct the installation instructions in the token vending machine readme.

### CODE REVIEW

File: examples/nodejs/lambda-examples/simple-get/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+This repo contains an example lambda, built using AWS CDK, that repeatedly calls get on a Momento cache.
+
+The primary use is to provide a base for testing Momento performance in a lambda environment. The lambda creates a Momento client, and then calls get on a hard-coded key 100 times, with a 100ms wait between calls. The metric logging middleware is enabled, so detailed information about each call is logged.
+
+## Prerequisites
+
+- Node version 14 or higher is required
+- To get started with Momento you will need a Momento Auth Token. You can get one from the [Momento Console](https://console.gomomento.com). Check out the [getting started](https://docs.momentohq.com/getting-started) guide for more information on obtaining an auth token.
+
+## Deploying the Token Vending Machine App
```
Comment (cprice404) [inline examples/nodejs/lambda-examples/simple-get/README.md:22]: this says TVM

File: examples/nodejs/lambda-examples/simple-get/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Simple Get Lambda
+
+This repo contains an example lambda, built using AWS CDK, that repeatedly calls get on a Momento cache.
+
+The primary use is to provide a base for testing Momento performance in a lambda environment. The lambda creates a Momento client, and then calls get on a hard-coded key 100 times, with a 100ms wait between calls. The metric logging middleware is enabled, so detailed information about each call is logged.
+
+## Prerequisites
+
+- Node version 14 or higher is required
+- To get started with Momento you will need a Momento Auth Token. You can get one from the [Momento Console](https://console.gomomento.com). Check out the [getting started](https://docs.momentohq.com/getting-started) guide for more information on obtaining an auth token.
```
Comment (cprice404) [inline examples/nodejs/lambda-examples/simple-get/README.md:20]: you should mention somewhere in here that you need to set the key ahead of time, probably with a cli example of how to set the key

File: examples/nodejs/lambda-examples/simple-get/infrastructure/lib/simple-get-stack.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      description: 'The Momento Auth Token that will be used to read from the cache.',
+      noEcho: true,
+    });
+
+    const authTokenSecret = new secrets.Secret(this, 'MomentoSimpleGetAuthToken', {
+      secretName: 'MomentoSimpleGetAuthToken',
+      secretStringValue: new cdk.SecretValue(momentoAuthTokenParam.valueAsString),
+    });
+
+    const tvmLambda = new lambdaNodejs.NodejsFunction(this, 'MomentoSimpleGet', {
```
Comment (cprice404) [inline examples/nodejs/lambda-examples/simple-get/infrastructure/lib/simple-get-stack.ts:23]: tvm

File: examples/nodejs/token-vending-machine/README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ These three required configuration variables live in the [config.ts](./lambda/to
 First make sure to start Docker and install the dependencies in the `lambda` directory, which is where the AWS Lambda code lives.
 
 ```bash
-cd lambda
+cd lambda/token-vending-machine
```
Comment (cprice404) [inline examples/nodejs/token-vending-machine/README.md:35]: tvm

Comment (cprice404) [reply examples/nodejs/token-vending-machine/README.md:35]: oh wait this one is actually correct just wasn't expecting in this PR, but it's a good change so 👍

**Review by cprice404 — DISMISSED** [review]
code looks good, a few cosmetic comments about README and leftover tvm variable names. feel free to merge and file another PR though if it's easier.

File: examples/nodejs/lambda-examples/simple-get/README.md (hunk 1 above)
Comment (pratik151192) [reply examples/nodejs/lambda-examples/simple-get/README.md]: we should probably fix this before pushing.. i know @cprice404 approved and i'm okay if you follow up with this

**Review by pratik151192 — APPROVED** [review]

**Pull request merged**